nncf/quantization/fake_quantize.py

```diff
@@ -122,11 +122,13 @@ def tune_range(
         fval = -left_border * s
         qval = fns.round(fval)
 
-    ra = fns.where(qval < level_high, qval / (qval - level_high) * right_border, left_border)
     with warnings.catch_warnings():
         # If `qval` is 0 `rb` will equal `right_border`, and we don't want to show an unnecessary division by 0 warning
+        # The same for (qval - level_high)
         warnings.simplefilter("ignore")
+        ra_then_result = qval / (qval - level_high) * right_border
```

Comment:
Are we sure that we hide only an `irrelevant` warning here? 
What was the reason for the initial warnings? 
What is the reason for hiding them?

Reply:
Thus, we're hiding the occurrence of errors because of `inf/nan` in the future, instead of reviewing the reasons for these errors. Am I right?

Reply:
no, because the ra and rb tensor will not have inf/nan because of fns.where condition, so the hiding these messages will not lead to any nan/inf values

Reply:
Thus, I see no reason to hide these calculations under the `warnings.catch_warnings()`.
For me, the existence of the `inf/nan` as the `ra_then_result` is the issue as in any other part.

Reply:
If you think so could you provide a solution to this better than I proposed and @nikita-savelyevv proposed in initial PR?

Reply:
The only better solution is to analyse the root cause of the `inf/nan` instead of hiding them.

Reply:
So no proposal, no solution from your side?

Reply:
Again and again. The proposal is to analyse the issues with `inf/nan` instead of hiding them. I don't know the root causes of the issue so I can't propose anything to fix it.
But I do not agree with just dropping them from the view.

Reply:
The root cause is that there are some zeros in denominator and these values are valid, because of that nan/inf gotten during the calculation but during filtration (fns.where) these values are filtered. 
So, there is nothing bad with these values and because of this they were hidden to not confuse anyone
What exactly do you think is bad in this pipeline?

Reply:
> some zeros in denominator and these values are valid

I don't get how it can be valid. Any examples?

Reply:
https://github.com/openvinotoolkit/nncf/pull/3081/files#diff-83c7c13863051f18f7e62d4d596695e2abb1d084ed6449c3fb78b42a86024d1fR27

Reply:
This test is irrelevant to me also. I wouldn't approve this either.

Reply:
@alexsu52 could you add your thoughts on this because you approved the initial PR - https://github.com/openvinotoolkit/nncf/pull/2415 ?